src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1378,13 +1361,9 @@ from appwrite.id import ID
 import os
 
 def main(context):
-    client = (
-        Client()
-            .set_endpoint("https://cloud.appwrite.io/v1")
-            .set_project(os.environ["APPWRITE_FUNCTION_PROJECT_ID"])
-            .set_key(os.environ["APPWRITE_API_KEY"])
-    )
-
+    client = Client()
+    # Set API key
+    client.set_key(context.req.headers.get('x-appwrite-key')) # API key is automatically provided by Appwrite
```

Comment:
we do `context.req.headers['x-appwrite-key']` elsewhere so let's use that syntax consistently.

```suggestion
    client.set_key(context.req.headers['x-appwrite-key']) # API key is automatically provided by Appwrite
```